src/ORM/DataObject.php

```diff
@@ -2485,6 +2489,65 @@ public function scaffoldFormFields($_params = null)
         return $fs->getFieldList();
     }
 
+    /**
+     * Scaffold a form field for selecting records of this model type in a has_one relation.
+     */
+    public function scaffoldFormFieldForHasOne(
+        string $fieldName,
+        ?string $fieldTitle,
+        string $relationName,
+        DataObject $ownerRecord
```

Comment:
Shouldn't this also have an &$includeInOwnTab param to for consistency with the others?  This would presumably have a default value of false,

Reply:
No, because there's never a good reason to have a has_one field in its own tab. The only reason the other relations have that is because it makes sense to have a gridfield in its own tab if you expect to have a lot of items in that relation.

Reply:
Note also that this method is being called from `DBForeignKey`, which doesn't have the context of whether the form scaffolder is "tabbed" or not - whereas the other two methods are being called from the form scaffolder directly.